doc/source/reliability/error_pad.rst

```diff
@@ -1,51 +1,47 @@
 .. _error_pad:
 
-==========================
- Error Code Compatibility
-==========================
+========================
+Error Code Compatibility
+========================
 
-|Percona Server| with |XtraDB| has error code incompatibilities with |MySQL| 5.5. It is important to maintain compatibility in the error codes used by the servers. For example, scripts that may be run on both servers could contain references to error codes.
+|Percona Server| has error code incompatibilities with |MySQL| 5.5. This could
```

Comment:
Contradiction: it simultaneously states that PS "has" error code incompatibilies, and further below describes how those incompatibilies are resolved, thus "hasn't"